rakelib/lint.rake

```diff
@@ -0,0 +1,28 @@
+namespace :lint do
```

Comment:
I expect those files in rakelib are gonna be copied over from Eve (that would serve as our source of truth for those utils) once generalized on all our repos?